### ISSUE TITLE
How can I find the model for the paper

### ISSUE THREAD

**Issue by Waynehfut** [surgical-vision/EndoVisPoseAnnotation#3]
Could you please help me with finding the model or the source code for the paper? The related paper says the model and code will provide here, but the only thing I can find is the data annotation. 
Thanks!

**Comment by wenxixi** [comment]
Hey I have the same issue here....  The paper says the model should be here but I cant find it. @duxiaofei283  Hey Xiaofei could you please help with this? Thanks!

**Comment by heizie** [comment]
@Waynehfut @wenxixi I don't think it is possible. I've alredy trained one with tf2 with these dataset. 

**Not surprisingly, I can't get the same result as the paper with NMS (same parameters)**. But just "almost there" accuracy with KMeans-clustering. 
And also the runtime can be optimised to 9ms inference + 10ms kmeans + 30ms mathching the points. with RTX 2080 ti+7700k.
I'm writing a other thesis with these solution for instrument tracking. But i've found this point-line method is not enough for detecting rotation of the instrument (as for our own flexibel intrument). That's why endovis start letting people to find out solutions for the segmentation's method. with these, i think it could use some CAD model and sticks it on the segmentation. similar to the thesis from her mentor Allan Max

============update=============================
can the author explain about, how she evaluate the pixel error? did she eveluate the only in 256x320 (Endovis)  images? or in the original size? It could make a huge difference

**Comment by heizie** [comment]
these are the testing results
[kmeans_bin.txt](https://github.com/surgical-vision/EndoVisPoseAnnotation/files/5200810/kmeans_bin.txt)
[kmeans_den.txt](https://github.com/surgical-vision/EndoVisPoseAnnotation/files/5200812/kmeans_den.txt)
[nms.txt](https://github.com/surgical-vision/EndoVisPoseAnnotation/files/5200813/nms.txt)
